src/main/resources/swagger/schemas/logged-request.yaml

```diff
@@ -7,22 +7,27 @@ properties:
   url:
     description: The path and query to match exactly against
     type: string
-    example: "/received-request/2"
+    example: "/received-request/2?foo=bar"
   absoluteUrl:
     description: The full URL to match against
     type: string
-    example: "http://localhost:56738/received-request/2"
+    example: "http://localhost:56738/received-request/2?foo=bar"
   headers:
     description: 'Header patterns to match against in the <key>: { "<predicate>": "<value>" } form'
     type: object
     example:
-      Connection: keep-alive
-      Host: localhost:56738
-      User-Agent: Apache-HttpClient/4.5.1 (Java/1.7.0_51)
+      Connection:
+        equalTo: keep-alive
```

Comment:
This isn't correct - we're logging the values of the headers not defining matchers. Suggest we just revert this to the previous version.

Reply:
Reverted!